src/src/radio.c

```diff
@@ -1643,11 +1713,8 @@ otError otPlatRadioTransmit(otInstance *aInstance, otRadioFrame *aFrame)
                             + SHR_DURATION_US);
         // Note - we need to call this outside of txCurrentPacket as for Series 2,
         // this results in calling the SE interface from a critical section which is not permitted.
-
 #if OPENTHREAD_RADIO && OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE == 1
         radioProcessTransmitSecurity(sTxFrame, sTxFrame->mIid);
-#else
-        radioProcessTransmitSecurity(sTxFrame, 0);
```

Comment:
The GSDK radio.c file does not have the #if check around the call to radioProcessTransmitSecurity.  Here's the snippet of text from the GSDK version.

```
        updateIeInfoTxFrame(sTxFrame->mInfo.mTxInfo.mTxDelayBaseTime + sTxFrame->mInfo.mTxInfo.mTxDelay + SHR_DURATION_US);
        // Note - we need to call this outside of txCurrentPacket as for Series 2,
        // this results in calling the SE interface from a critical section which is not permitted.
        radioProcessTransmitSecurity(sTxFrame, sTxFrame->mIid);
#endif // OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2

```

Reply:
I see. Thanks.